x/auth/client/cli/tx_multisign.go

```
@@ -146,11 +147,32 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) error {
 
 		sigOnly, _ := cmd.Flags().GetBool(flagSigOnly)
 
-		json, err := marshalSignatureJSON(txCfg, txBuilder, sigOnly)
+		aminoJSON, err := cmd.Flags().GetBool(flagAmino)
```

we can ignore the error IMHO